src/Handler/DownloadHandler.php

```diff
@@ -68,9 +68,12 @@ private function createDownloadResponse($stream, string $filename, ?string $mime
             \stream_copy_to_stream($stream, \fopen('php://output', 'wb'));
         });
 
+        $filename = str_replace(['%', '/', '\\'], '', $filename);
+
         $disposition = $response->headers->makeDisposition(
             $forceDownload ? ResponseHeaderBag::DISPOSITION_ATTACHMENT : ResponseHeaderBag::DISPOSITION_INLINE,
-            $filename
+            $filename,
+            filter_var($filename, FILTER_SANITIZE_STRING, FILTER_FLAG_STRIP_HIGH)
```

Comment:
this function must be called from root namespace (as well as any other functions)